testng-eclipse-plugin/src/main/org/testng/eclipse/TestNGPluginConstants.java

```diff
@@ -31,5 +31,14 @@
   public static final String S_VIEW_TITLE_SHOW_CASE_NAME = "showCaseNameOnViewTitle";
   public static final String S_APPEND_FAVORITE_STATIC_IMPORT = "appendFavoriteStaticImport";
 
+  /** The name of the TestNG DTD. */
+  public static final String TESTNG_DTD = "testng-1.0.dtd";
+
+  /** The URL to the deprecated TestNG DTD. */
+  public static final String DEPRECATED_TESTNG_DTD_URL = "http://beust.com/testng/" + TESTNG_DTD;

```

Comment:
@krmahadevan, not sure what's the reason you made these field as private in commit https://github.com/cbeust/testng/commit/40ea46dfecafb58017b221f57e90942e361fcb8d#diff-a14c30b8e000eb785a5740db54ecbd56

anyway, i just have a copy of them in Testng Eclipse plugin to make code compiler work

Reply:
@missedone - No specific reason. I must have given into the recommendations from IntelliJ and made it private.. My bad! Please let me know which fields need to be exposed and I will have them reverted in TestNG `master`

Reply:
pls revert `DEPRECATED_TESTNG_DTD_URL` and `TESTNG_DTD_URL`, thx

Reply:
@missedone PR https://github.com/cbeust/testng/pull/2131 raised for this.